source/adapters/hip/device.hpp

```diff
@@ -97,3 +91,20 @@ struct ur_device_handle_t_ {
 };
 
 int getAttribute(ur_device_handle_t Device, hipDeviceAttribute_t Attribute);
+
+namespace {
+/// Scoped Device is used across all UR HIP plugin implementation to activate
+/// the native Device on the current thread. The ScopedDevice does not
+/// reinstate the previous device as all operations in the hip adapter that
```

Comment:
Looks like this was copied from an older comment but we have `HIP` and `hip` in the same comment. I think `HIP` is best?

Reply:
Thanks, I've used HIP consistently now.